CMakeLists.txt

```diff
@@ -512,13 +512,17 @@ endif()
 option(WITH_SYSTEM_BOOST "require and build with system Boost" OFF)
 
 set(BOOST_COMPONENTS
-	thread system regex random program_options date_time iostreams coroutine context)
+	thread system regex random program_options date_time iostreams)
 set(BOOST_HEADER_COMPONENTS container)
 
 if(WITH_MGR)
 	list(APPEND BOOST_COMPONENTS python)
 endif()
 
+if(WITH_RADOSGW_BEAST_FRONTEND)
```

Comment:
@smithfarm debian supports mips and mipsel, so for the sake of completeness, we can list it in `debian/rules` like
```makefile
ifeq (,$(filter $(DEB_HOST_ARCH), arm armel armhf arm64 i386 amd64 mips mipsel powerpc ppc64))
  extraopts += -DWITH_RADOSGW_BEAST_FRONTEND=ON
else
  extraopts += -DWITH_RADOSGW_BEAST_FRONTEND=OFF
endif
```

i understand if you intent to keep clean and minimal. but i just wanted to the packaging to be a better reference for our downstreams or potential users.